src/libs/OptionsListUtils.js

```diff
@@ -686,6 +686,7 @@ function getSidebarOptions(
     };
     if (priorityMode === CONST.PRIORITY_MODE.GSD) {
         sideBarOptions = {
+            prioritizePinnedReports: true,
```

Comment:
If this is the default for all modes then we don't need it as part of `sidebarOptions`. we can directly pass it to `getOptions`.

Reply:
okay.